### PR TITLE
Added a note to highlight  difference between -AzureAD and -AzureADMS cmdlets

### PR DESCRIPTION
**Pull request by msewaweru**

### CODE REVIEW

**Comment by billmath** [comment]
Docs Build status updates of commit _[d186cc3](https://github.com/msewaweru/azure-docs-powershell-azuread/commits/d186cc36b0e232756511c57b3f0b1d5a9e596ec1)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-59a43b14)

<a id="59a43b14"></a>
###
 - **[Error-RuningBuildFailed]** `````Some unexpected errors happened when running build, please open a ticket in https://aka.ms/SiteHelp and include the error report for our team to troubleshoot`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C8%5C16%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202108160849409845-680%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=pRPVb8OZ%2B76yXaj%2B9TEhoRXgHMqFZzgJwgd1PETHjoY%3D&st=2021-08-16T08%3A45%3A18Z&se=2021-09-16T08%3A50%3A18Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=master) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by billmath** [comment]
Docs Build status updates of commit _[d186cc3](https://github.com/msewaweru/azure-docs-powershell-azuread/commits/d186cc36b0e232756511c57b3f0b1d5a9e596ec1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-conceptual/azureadps-2.0/install-adv2.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/overview-update/docs-conceptual/azureadps-2.0/install-adv2.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/install-adv2?view=azureadps-2.0&branch=pr-en-us-680) (azureadps-2.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C8%5C16%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202108160850415974-680%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=gp63I7NTjrPuNWNAlVDZU7dL8AaBIfxS1QeMjIhfN8Q%3D&st=2021-08-16T08%3A47%3A41Z&se=2021-09-16T08%3A52%3A41Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by msewaweru** [comment]
Fixes #607.

**Comment by msewaweru** [comment]
@CelesteDG , please review this.

File: docs-conceptual/azureadps-2.0/install-adv2.md [hunk 1]
```diff
@@ -123,6 +123,9 @@ Connect-AzureAD -Credential $AzureAdCred
 The first command prompts for credentials and stores them as $AzureAdCred.
 The next command uses those credentials as $azureadcred to connect to the service.
 
+> [!Note]
+> The Azure AD and Azure AD Preview modules comprise of cmdlets with different naming conventions i.e. `-AzureAD` and `-AzureMS`. The `-AzureAD` cmdlets connect to the Azure AD Graph endpoint `https://graph.windows.net` while the `-AzureADMS` make calls to the Microsoft Graph endpoint `graph.microsoft.com`. Authentication is handled silently when you change the calls from one endpoint to another and you are not prompted for the credentials again.
```
Comment (CelesteDG) [inline docs-conceptual/azureadps-2.0/install-adv2.md:127]: @msewaweru In the naming convention, you call out -AzureAD and -AzureMS. However, in the additional info the second convention is -AzureADMS. Note the addition of 'AD'. Please reconcile.

**Review by CelesteDG — CHANGES_REQUESTED** [review]
@msewaweru - Please see my comments. Thank you.

**Comment by billmath** [comment]
Docs Build status updates of commit _[435e444](https://github.com/msewaweru/azure-docs-powershell-azuread/commits/435e444569b225d2f8af2970dd10622c4f570fa0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-conceptual/azureadps-2.0/install-adv2.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/overview-update/docs-conceptual/azureadps-2.0/install-adv2.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/install-adv2?view=azureadps-2.0&branch=pr-en-us-680) (azureadps-2.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C8%5C19%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202108190929417692-680%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=EFYxSer0LTX7nvPumuSmlA5yKVUvMLA3psa53kObF54%3D&st=2021-08-19T09%3A25%3A40Z&se=2021-09-19T09%3A30%3A40Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by msewaweru** [comment]
Hi @CelesteDG, correction made. Thanks.

**Pull request merged**